projects/fusion-ui/components/loader/v4/loader.component.ts

```diff
@@ -0,0 +1,22 @@
+import {ChangeDetectionStrategy, Component, Input} from '@angular/core';
+import {CommonModule} from '@angular/common';
+
+@Component({
+    selector: 'fusion-loader',
+    standalone: true,
+    imports: [CommonModule],
+    templateUrl: './loader.component.html',
+    styleUrls: ['./loader.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class LoaderComponent {
+    @Input() set strokeWidth(value: number) {
+        this._strokeWidth = value;
+    }
+
+    get strokeWidth(): number {
+        return this._strokeWidth ?? 2;
```

Comment:
We don't want to put hard-coded values here no? lets put it on a constant file

Reply:
Done.